cylc/flow/config.py

```diff
@@ -211,6 +212,17 @@ def __init__(
         if icp_str is not None:
             self.cfg['scheduling']['initial cycle point'] = icp_str
 
+        if 'cylc' not in self.cfg:
+            self.cfg['cylc'] = {}
+
+        if self.cfg['cylc'].get('cycle point time zone') is None:
+            # Get the original suite run time zone if restart:
+            cp_tz_str = getattr(self.options, 'cp_tz', None)
+            if cp_tz_str is None:
+                # Not a restart
+                cp_tz_str = get_local_time_zone_format()
+            self.cfg['cylc']['cycle point time zone'] = cp_tz_str
+
```

Comment:
I think we allow `cycle point time zone` to be set in the `[cylc]` section of the global configuration as well so will have to check the value there as well.

Note the scheduler has a function which loads `[cylc]` items by looking at the suite config first and the global config second.

Reply:
`cycle point time zone` isn't in globalcfg.py but it is in suite.py. If I understand correctly I just need to move that block from after
https://github.com/cylc/cylc-flow/blob/8a9043187eb7948674de86f53aa039908a92ef0c/cylc/flow/config.py#L186-L192

to after
https://github.com/cylc/cylc-flow/blob/8a9043187eb7948674de86f53aa039908a92ef0c/cylc/flow/config.py#L306-L309

Reply:
Should the fact that it isn't in globalcfg be raised in a new issue? Or is that okay?

Reply:
If it's not in the global config then we are good to go with no further changes!

Reply:
After thinking about this I do think it's worth moving the block as described above, otherwise if we decide to set a default cycle point time zone it will never be used